errors/manifest.json

```diff
@@ -712,7 +712,11 @@
         },
         {
           "title": "middleware-dynamic-wasm-compilation",
-          "path": "/errors/middleware-dynamic-wasm-compilation.md"
+          "path": "/errors/edge-dynamic-code-evaluation.md"
```

Comment:
We should leave the previous file in place so older Next.js versions don't 404 for this link